cli-feed-v4.json

```diff
@@ -1752,6 +1757,306 @@
+          "sha2": "0187b0e6101a3b9511f0ea58be24026e2836a6b79680b8d1508a245369e9dbbf",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.osx-x64.3.0.3528.zip",
+          "sha2": "a6137a79a14a5bd55d7fafe3c05200db7c767c9ca0e17cbca9f5d24c273100ea",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.min.win-x64.3.0.3528.zip",
+          "sha2": "2528ea9cee707cc4787f201e1705b4264cc2b8b8d67a3fabdfad3ca1b5a06da4",
+          "size": "minified",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.win-x86.3.0.3528.zip",
+          "sha2": "83478b84d66ed52e508e12a926666e1ad85fb9b7e7e091820b52f7ae3aae5bcd",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.min.win-x86.3.0.3528.zip",
+          "sha2": "802e2ca6a355099ea7491beec932f80ae6fce8968b8f94a40cbb4dd36c3490fc",
+          "size": "minified",
+          "default": "true"
+        }
+      ]
+    },
+    "4.0.0": {
+      "templates": "<TODO>",
+      "workerRuntimes": {
+        "dotnet": {
+          "net6": {
+            "displayInfo": {
+              "displayName": ".NET 6",
+              "hidden": false,
+              "displayVersion": "v4",
+              "targetFramework": ".NET",
+              "description": "LTS"
+            },
+            "capabilities": "net6",
+            "sdk": {
+              "name": "Microsoft.NET.Sdk.Functions",
+              "version": "<TODO>"
+            },
+            "default": true,
+            "localEntryPoint": "func.exe",
+            "targetFramework": "netcore6.0",
+            "itemTemplates": "<TODO>",
+            "projectTemplates": "<TODO>",
+            "projectTemplateId": {
+              "csharp": "<TODO>"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet:6.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~4",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v6.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET|6.0"
+            }
+          },
+          "net5-isolated": {
```

Comment:
V4 .NET 5 Isolated

Reply:
should add 'net6' to the capabilities list, as the func.exe will run net6.